README.md

```diff
@@ -19,14 +20,15 @@ Installation
 Install AWS CLI Tools:
 
 	$ pip install awscli
+	$ sudo apt-get install php5-cli
 	
 Install aws-profile:
 
-	$ curl -sSL https://raw.github.com/jaymecd/aws-profile/master/install.sh | sh
+	$ curl -sSL https://raw.github.com/Home24/aws-profile/master/install.sh | sh
 	
 Or via clone:
 
-	$ git clone https://github.com/jaymecd/aws-profile.git
+	$ git clone https://github.com/Home24/aws-profile.git
 	$ cp aws-profile/aws-* /usr/local/bin
 	
 Update resource file
```

Comment:
It's nice to export the `AWS_CONFIG_FILE` environment variable, because the default fallback value in the script is set on a new shell only when `aws-profile ...` command is run, otherwise any `aws` command throws an error: `ERROR: Env AWS_CONFIG_FILE must be set`.